feature/interface/ip/ipv6_link_local/otg_tests/ipv6_link_local_test/ipv6_link_local_test.go

```diff
@@ -255,9 +255,17 @@ func verifyLinkLocalTraffic(t *testing.T, dut *ondatra.DUTDevice, ate *ondatra.A
 	p1 := dut.Port(t, "port1")
 	beforeInPkts := gnmi.Get(t, dut, gnmi.OC().Interface(p1.Name()).Counters().InPkts().State())
 	ate.OTG().StartTraffic(t)
-	time.Sleep(15 * time.Second)
+	if deviations.InterfaceCountersUpdateDelayed(dut) {
```

Comment:
A better pattern is to use gnmi.Watch.Await,  for example:

https://github.com/openconfig/featureprofiles/blob/be17a31486f158e46f857fbe6e9822becb84c8bf/feature/interface/singleton/otg_tests/singleton_test/singleton_test.go#L448-L461

See also the ondatra guidance for how to use Await to reach a goal state:

https://github.com/openconfig/ondatra/blob/b07a95699a4eed8aa606762baadc0a96074c4394/gnmi/gnmi.go#L42-L86

Reply:
done - please check my change now.

Reply:
Changes looks good now.